LanguageFeatures/Patterns/if_case_A02_t02.dart

```diff
@@ -0,0 +1,300 @@
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testVariable(1));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testVariable(-1));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testList1([1, 42]));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testList1([2, 42]));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testList1([1, 0]));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testList1([1, 42, 0]));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testList2([1, -42]));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testList2([2, -42]));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testList2([1, 0]));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testList2([1, -42, 0]));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testList3([1, 2, 3]));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testList3([2, 2]));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testList3([1, 0]));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testList3([1, -42, 0]));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4},
+        testMap1({"key1": 1, "key2": 2}));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4},
+        testMap1({"key1": 2, "key2": 2}));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4},
+        testMap1({"key1": 1, "key2": -2}));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4},
+        testMap1({"key1": 1, "key2": 2, "key3": 3}));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4},
+        testMap2({"key1": 1, "key2": -2}));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4},
+        testMap2({"key1": 2, "key2": -2}));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4},
+        testMap2({"key1": 1, "key2": 2}));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4},
+        testMap2({"key1": 1, "key2": -2, "key3": 3}));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testRecord1((1, 2)));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testRecord1((2, 2)));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testRecord1((1, -2)));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4},
+        testRecord2((1, 2, n: 1)));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4},
+        testRecord2((2, 2, n: 1)));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4},
+        testRecord2((1, 2, n: -1)));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4},
+        testRecord2((1, n: 1)));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4},
+        testRecord2((1, 2, n: 1, n2: 2)));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testObject1(Square(3)));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testObject1(Circle(3)));
+    Expect.mapEquals({"key1": 1, "key3": 3, "key4": 4}, testObject1(Square(1)));
+    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testObject1(Circle(3)));
```

Comment:
```suggestion
    Expect.mapEquals({"key1": 1, "key2": 2, "key4": 4}, testObject2(Circle(3)));
```

Same fixes again in lines below.